### DISCUSSION TITLE
How do I only update one folder in a repo?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21799] [New to GitHub]
<p>I have found questions similar to this one, but not quite the same. I have a repo called RepoA. Inside of this repo I have several folders Folder1, Folder2, etc. I have multiple developers working on this repo, but I don’t want them to have access to Folder1. I want developers to only be able to download Folder2 for example and only be able to push Folder2 without even needing to download or see the rest of the folders.</p>
<p>Thanks</p>
<p>I hope I worded my question correctly if clarification is needed I will provide it.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That is not possible with one git repository. You’ll have to split the data into multiple repositories if you want to control access separately.</p>
<p>Submodules might be useful if you need version control for the combination, but if that makes sense depends on the purpose of your data.</p>
